### PR TITLE
Feat kpis

### PR DESCRIPTION
**Pull request by mohityadav766**
### Describe your changes :

Feature #8472
#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  {
+    "name": "EntityWithTier",
+    "displayName": "Entity with Tier",
+    "fullyQualifiedName": "EntityWithTier",
+    "description": "Entities should have Tiers",
+    "metrics": [
+      {
+        "name": "percentComplete",
+        "dataType": "NUMBER",
```
Comment (TeddyCr) [inline openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json:50]: Should that be `PERCENTAGE`?

File: openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  {
+    "name": "EntityWithDescription",
+    "displayName": "Entity with Description",
+    "fullyQualifiedName": "EntityWithDescriptions",
+    "description": "Entities should have Descriptions",
+    "metrics": [
+      {
+        "name": "percentComplete",
+        "dataType": "NUMBER",
```
Comment (TeddyCr) [inline openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json:31]: Should that be `PERCENTAGE`?

File: openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  {
+    "name": "EntityWithOwnership",
+    "displayName": "Entity with Ownership",
+    "fullyQualifiedName": "EntityWithOwnership",
+    "description": "80 percent of Entities should have Owners",
+    "dateInsightChart": "refrence to the chart for this kpi",
+    "target": [
+      {
+        "name": "completedDescriptionFraction",
+        "dataType": "PERCE",
```
Comment (TeddyCr) [inline openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json:11]: Should that be `PERCENTAGE`?

**Review by TeddyCr — COMMENTED** [review]
@mohityadav766 what do you think about moving the `basic.json` and `kpi.json` to the `dataInsight` folder?

File: openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json (hunk 1 above)
Comment (mohityadav766) [reply openmetadata-service/src/main/resources/json/data/kpi/kpiDefinitionsData.json:50]: this needs to be removed , don't need this json actually

**Comment by mohityadav766** [comment]
> @mohityadav766 what do you think about moving the `basic.json` and `kpi.json` to the `dataInsight` folder?

@TeddyCr moved to suggested location

**Review by harshach — APPROVED** [review]

**Pull request merged**